web/js/codeworld.js

```diff
@@ -248,6 +248,23 @@ function help(doc) {
     });
 }
 
+function editorHelp(doc) {
+    var helpText = "<h3>Editor Shortcuts</h3>" +
+        "<div id='keyboard-shortcuts'><table><tbody>" +
+        "<tr><td>Ctrl + Space </td><td> Autocomplete</td></tr>" +
+        "<tr><td>Ctrl + Up </td><td> Zoom In </td></tr>" +
+        "<tr><td>Ctrl + Down </td><td>  Zoom Out </td></tr>" +
+        "</tbody></table></div>";
```

Comment:
There are a lot more keys here!

Ctrl-A: select all
Alt-Left: go to start of line
Alt-Right: go to end of line
Ctrl-Home: go to start of doc
Ctrl-End: go to end of document
Ctrl-D: delete a line
Ctrl-Left: go left a word
Ctrl-Right: go right a word
Ctrl-Backspace: delete the previous word
Ctrl-Delete: delete the next word
Ctrl-F: start a new search
Ctrl-G: search for next occurrence
Ctrl-Shift-G: search for previous occurrence
Shift-Ctrl-F: replace
Shift-Ctrl-R: replace all
Ctrl-S: Save
Ctrl-Z: undo
Ctrl-Y or Ctrl-Shift-Z: redo
Ctrl-U: undo selection
Alt-U or Ctrl-Shift-U: redo selection
Tab or Ctrl-]: indent
Shift-Tab or Ctrl-[: un-indent
Ctrl-I: reformat (but *only* at code.world/haskell, not at the main CodeWorld URL)